testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc="nodejs bash"
+makedepends="linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
+	zerotier-one.initd
+	"
+
+builddir="$srcdir"/ZeroTierOne-$pkgver
+build() {
+	cd "$builddir"
+
+	# build base binaries
+	make DESTDIR="$builddir" one || return 1
```

Comment:
Why do you set DESTDIR when you are building?
Please also build docs within the build function, ie build all.